tests/integration/test_install_basic.py

```diff
@@ -1,50 +1,23 @@
 import os
-import mock
-
 from pathlib import Path
 from tempfile import TemporaryDirectory
 
 import pytest
 
```

Comment:
By removing flaky we aren't trying again on flaky internet connections. I believe this was the original intent.
Does it really slow things that badly?

Reply:
Well my problem with it is the tests seem randomly annotated with flaky -- Its almost like every test should be labeled flaky -- in which case let's not annotate them and just always retry it once.  But the other side of my brain thinks -- well we are either connecting to pypi which is fronted by Fastly, or our private pypi, or github.   I haven't really seen flaky ever kick in a way that isn't become someone broke a test and so it retires it -- but maybe some people like it?  I personally think it adds confusion.